### PR TITLE
[AND-4661] Update android 7.0 SDK sample app code structure.

### PR DESCRIPTION
**Pull request by anton-blashchuk-zoolatech**
separated Kotlin and Java projects while retaining history
https://vungle.atlassian.net/browse/AND-4661

### CODE REVIEW

File: Samples/android/Kotlin/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Getting Started
+Please refer to https://support.vungle.com/hc/en-us/articles/360002922871
+
+Vungle SDK for Android
+=======================
+
+**Version 6.12.0**
```
Comment (felixzhang0703) [inline Samples/android/Kotlin/README.md:7]: Update to 7.0.0

File: Samples/android/Kotlin/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## Getting Started
+Please refer to https://support.vungle.com/hc/en-us/articles/360002922871
```
Comment (felixzhang0703) [inline Samples/android/Kotlin/README.md:2]: Update to 7.0 document. Still in progress.

File: Samples/android/Kotlin/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+**Version 6.12.0**
+
+Welcome to the Vungle SDK which has been battle-tested to unlock amazing monetization opportunities for you. The Vungle SDK enables the very best creatives in mobile advertising.
+ 
+## Attributions
+| Component                       | Version | Description                                                                               | License                                                                                            |
+|---------------------------------|---------|-------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| Android X (Jetpack)             | 1.1.0   | Core and LocalBroadcastManager Android X libraries from Jetpack                           | [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)                                          |
+| Google Gson                     | 2.8.6   | Java library to serialize and deserialize Java objects to JSON                            | [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)                                          |
```
Comment (felixzhang0703) [inline Samples/android/Kotlin/README.md:15]: update to kotlinx-serialization-json 1.4.1

File: Samples/android/Kotlin/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+| OkHttp                          | 3.14.9  | An HTTP+HTTP/2 client for Android and Java applications                                   | [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)                                          |
```
Comment (felixzhang0703) [inline Samples/android/Kotlin/README.md:16]: update okhttp version to 4.10.0

File: Samples/android/Kotlin/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,256 @@
+## Changelog
+
+### VERSION 6.12.0 (August 2, 2022)
```
Comment (HuFangshuai) [inline Samples/android/Kotlin/CHANGELOG.md:3]: Add TODO for 7.0.0

**Review by HuFangshuai — APPROVED** [review]
lgtm

**Pull request merged**